packages/locale/lang/zh-tw.ts

```diff
@@ -76,19 +98,33 @@ export default {
       pagesize: '項/頁',
       total: '共 {total} 項',
       pageClassifier: '頁',
+      deprecationWarning:
+        'Deprecated usages detected, please refer to the el-pagination documentation for more details',
```

Comment:
```suggestion
        '你使用了一些已被廢棄的用法，請參考 el-pagination 的官方文檔',
```
Are you forgetting to edit here? I do not know traditional Chinese, the specific changes are for reference only

Reply:
@KimYangOfCat Oh, thanks for your kind remainder!
I have updated the translation.

https://github.com/element-plus/element-plus/blob/3b56da2e07f4e23d6e79005735958c9ae9567423/packages/locale/lang/zh-tw.ts#L101-L102